test/erlcloud_s3_tests.erl

```diff
@@ -60,7 +60,7 @@ error_handling_no_retry() ->
     Response = {ok, {{500, "Internal Server Error"}, [], <<"TestBody">>}},
     meck:expect(erlcloud_httpc, request, httpc_expect(Response)),
     Result = erlcloud_s3:get_bucket_policy("BucketName", config()),
-    ?_assertEqual({error,{http_error,500,"Internal Server Error",<<"TestBody">>}}, Result).
+    ?_assertEqual({error,{http_error,500,"Internal Server Error",<<"TestBody">>,[]}}, Result).
```

Comment:
I see this test is updated, but I don't see any test case for the new functionality being introduced.